lib/ui/semantics.dart

```diff
@@ -419,6 +419,7 @@ class SemanticsUpdateBuilder extends NativeFieldWrapperClass2 {
     String value,
     String increasedValue,
     String decreasedValue,
+    String routeName,
```

Comment:
RouteName needs an entry in the doc comment above.

Reply:
Fixed ish